server/modules/analysis/manager/processingChainManager.js

```diff
@@ -214,6 +214,14 @@ const _updateCalculationIndexes = async (user, surveyId, step, t) => {
 
 export { countChainsBySurveyId, fetchChainsBySurveyId, fetchChainByUuid } from '../repository/processingChainRepository'
 
+export const fetchStepData = async (survey, cycle, stepUuid) => {
+  const surveyId = Survey.getId(survey)
+  let step = await ProcessingStepRepository.fetchStepSummaryByUuid(surveyId, stepUuid)
+  const calculations = await ProcessingStepCalculationRepository.fetchCalculationsByStepUuid(surveyId, stepUuid)
```

Comment:
use ```ProcessingStepRepository.fetchStepsAndCalculationsByChainUuid```

Reply:
but we need to fetch only one step, not all the steps...